add work section

RichardKaranuMbuti opened the pull request:
added work section of mobile portfolio

Ibizugbe reviewed (commented):
Hi @RichardKaranuMbuti 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that lacks a descptive summary. kindly add a descriptive summary and itemize the features you added to the branch😉 afer doing that, kindly resubmit again. we cant wait to review your amazing project🤗

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

RichardKaranuMbuti commented:
@Ibizugbe 
Thank you for the review,I dont understand what you mean,should  I add  a descriptive summary in a readme file or as a commit/comment message

RichardKaranuMbuti commented:
added work section with 3 work samples and the links
Included a section about me below the work section
Added a section on my skills 
Included the section on language frameworks am proficient in

NduatiKagiri reviewed (changes requested):
Hi @RichardKaranuMbuti ,

# CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights ✔️ 
✅ GitHub Flow observed
✅ No Linters errors

## Required Changes ♻️

- [ ] It seems you are misusing the `section` element in your code. Please note that this element is used to wrap a whole section. For example, in this project milestone, you should only have 2 `section` elements, one that wraps the whole of the Works section, and one for the About section. You can consider using the `div` element in the places where you are wrongly using `section`. This ensures perfect UI/UX.
- [ ] You have not used the right padding and margins in your Works section.
-   Your design:
     ![Uploading karanu1.JPG…]()

-   Proposed Figma design:
    ![karanu2](https://user-images.githubusercontent.com/87394385/212491511-dcc476b0-18c0-421f-bf60-e13103a3fb18.JPG)

-   Kindly make sure you fix the headings that are floating on top of the images and ensure you also apply the right fonts, font sizes, padding, and margins as proposed in the Figma design.
-   I have taken a screenshot of only one card, but the mentioned errors are there in all the cards in the work section. Kindly fix the errors on all the cards.

- [ ] _Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
